app/tsconfig.json

```diff
@@ -12,9 +16,9 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve",
+    "jsx": "react-jsx",
```

Comment:
Curious: what does this change do?

Reply:
I'm honestly not fully clear, even after reading the [documentation](https://www.typescriptlang.org/docs/handbook/jsx.html)... But the tests would fail when I kept this as "preserve"...

Reply:
See https://github.com/navapbc/template-application-nextjs/pull/9